packages/moonstone/SlotItem/SlotItem.module.less

```diff
@@ -28,7 +28,7 @@
 			> * {
 				margin-top: 0;
 				margin-bottom: 0;
-				vertical-align: top;
+				vertical-align: center;
```

Comment:
What are the implications to other uses of `SlotItem` by making this change?
The comment a few lines above explicitly calls-out vertical alignment, so I have to assume that `vertical-align: top;` was written out _on purpose_.

Reply:
If it resized the image from app side, it was not centered.

Reply:
[https://jira2.lgsvl.com/secure/attachment/474561/%EC%84%A0%ED%83%9D%20%EC%98%81%EC%97%AD_009.png](url)

Reply:
@Djspaceg , @ryanjduffy , Could you please look at the above screenshot?
Is there any method?